sh/src/helm-template

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```

Comment:
I suggest organzing this repo by function instead:

helm/
--  Dockerfile
--  helmp-template.sh
-- test

Reply:
Changed, also removed test data based on below comment.